src/current/cockroachcloud/ccloud-get-started.md

```diff
@@ -397,6 +397,15 @@ If the organization is incorrect:
 
 {{site.data.alerts.end}}
 
+### Skip the IP allowlist check when connecting to your cluster
+
+By default, the `ccloud cluster sql` command will allow connections only from IP addresses in your cluster's [allowlist](network-authorization.html#ip-allowlisting). Use the `--skip-ip-check` flag to disable the client-side IP allowlist check:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+ccloud cluster sql dim-dog --skip-ip-check
```

Comment:
Nit, but is `dim-dog` just an autogenerated cluster name? Maybe choose something a little more positive?

Reply:
we use dim-dog on the rest of the page and 68 times in 26 file right now so i'm going to leave it for consistency, but will open a separate pr to change